tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/GraphOperationBuilder.java

```diff
@@ -83,16 +85,24 @@
    */
   @Override
   public GraphOperation build() {
+    scope.apply(this);
```

Comment:
So this adds all control dependencies from the scope to everything built by that ops? Isn't this a behaviour change?

Reply:
It was always applied manually in the generated code, i.e:
```java
OpBuilder builder = scope.env().opBuilder("", "");
builder.addInput(..);
builder = scope.apply(builder);
```
and not doing so would lead to bugs with name collisions (there was one in `Constant` at one point).

It would be a behavior change if you used the op builder manually without every applying the scope, but that would usually be a bug and wasn't the case in any of the non-test usages (Constant and ConcreteFunction, and ConcreteFunction's will be removed soon).